app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class RegistrationsController < Devise::RegistrationsController
+  def new
+    super
+  end
+
+  def create
+    super
+  end
+
+  def update
+    super
+  end
```

Comment:
You can remove all these 3 methods now. They are not needed. Make sure you run all tests before you push up.